src/Atmos/Model/radiation.jl

```diff
@@ -43,24 +43,28 @@ struct StevensRadiation{FT} <: RadiationModel
   "F₁ Radiative flux parameter [W/m^2]"
   F_1::FT
 end
-vars_integrals(m::StevensRadiation, FT) = @vars(∂κLWP::FT)
-function integrate_aux!(m::StevensRadiation, integrand::Vars, state::Vars, aux::Vars)
+vars_integrals(m::DYCOMSRadiation, FT) = @vars(∂κLWP::FT)
```

Comment:
Please change `∂κLWP` to the clearer `attenuation_coeff` or `attenuation_coefficient` globally.